wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js

```diff
@@ -119,7 +129,7 @@ define(
 					elementData.reactButton = elBySel(this._options.buttonSelector, element);
 				}
 				
-				if (elementData.reactButton === null || elementData.reactButton.length === 0) {
+				if (elementData.reactButton === null || elementData.reactButton.length === 0) {
```

Comment:
This change is fun. For some reason this previously was a non-breakable space, but the code worked just fine?

Reply:
It was a strange finding, PhpStorm displayed a box with `NBSP` inside and I was a bit confused at first.